R/utils.R

```diff
@@ -229,6 +242,30 @@ adjustToViewport <- function(x, viewport) {
   return(x)
 }
 
+#' Filter Data Table to Geolocation Viewport
+#' 
+#'
+#' This function will filter a data.table to the specified 
+#' latitude and longitude values.
+#' @param .dt data.table with columns for latitude and longitude
+#' @param latitude Character vector specifying name of latitude column
+#' @param longitude Character vector specifying name of longitude column
+#' @param viewport List of lists specifying the geolocation viewport
```

Comment:
also may be helpful to specify list names that are expected, like xMin and left/right.